package.json

```diff
@@ -10,16 +10,16 @@
     "dist"
   ],
   "scripts": {
-    "test": "tsc --noEmit && vitest --run && vitest -c .vitest.config/jsx-runtime-default.ts --run && vitest -c .vitest.config/jsx-runtime-dom.ts --run",
+    "test": "tsc --noEmit && vitest --run",
```

Comment:
`vitest --run` will run all projects, including all runtime tests.

The equivalent command for just the jsx runtime tests would be..

```sh
vitest --project jsx-runtime-default --project jsx-runtime-dom --run
```